sklearn/cluster/tests/test_optics.py

```diff
@@ -216,237 +215,167 @@ def test_cluster_sigmin_pruning(reach, n_child, members):
     assert_array_equal(members, root.children[0].points)
 
 
-def test_reach_dists():
+def test_compare_to_ELKI():
+    # Expected values, computed with (future) ELKI 0.7.5 using:
```

Comment:
Maybe simply using things like ``dev version``? It seems strange. I'll take care of it after ELKI is released so you don't need to make unnecessary promise here.